components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -567,6 +569,7 @@ public static String buildCacheKeyStringForAuthzCode(String clientId, String aut
      * @param authorizedUser
      * @return
      */
+    @Deprecated
```

Comment:
Please add a reason to deprecate and the new method to be used instead.

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1128